internal/website/docs/next/reference/helpers/intro.mdx

```diff
@@ -0,0 +1,10 @@
+---
+slug: /next/reference/helpers
+title: Helpers
```

Comment:
How should we define a "Helper"? Should we move the other functions (ex. `is404`, `withFaust`, `FaustProvider`, etc) here as well?